_tuning-your-cluster/availability-and-recovery/remote-store/remote-cluster-state.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: default
+title: Remote Cluster State
+nav_order: 5
+parent: Remote-backed storage
+grand_parent: Availability and recovery
+---
+
+# Remote Cluster State
+
+Introduced 2.10
+{: .label .label-purple }
+
+Remote Cluster State protects against any cluster state metadata loss resulting due to quorum loss (permanently losing majority of cluster manager nodes) in the cluster.
+
+Cluster State in an internal data structure which contains the metadata of the cluster along with other information. 
+The metadata includes details about index metadata like settings, mappings, active copies of the shards, cluster level settings, aliases, templates and data streams etc. 
+This metadata is managed by the elected cluster manager node and is essential for proper functioning of the cluster. When the cluster loses majority of the cluster manager nodes permanently, lets say 2 out of 3 cluster manager nodes are lost, then the cluster can experience data loss as there are no guarantees that latest cluster state metadata is present in the surviving nodes. Today, cluster durability is the function of the cluster manager node storage. And, persisting the state to remote provides better durability guarantees.
+
+When remote cluster state feature is enabled, the cluster metadata will be published to a remote repository configured in the cluster. Note that, currently only index metadata will be persisted to remote store in OpenSearch 2.10.
```

Comment:
we should mention about global metadata persistence as well since in 2.11 we release that too

Reply:
Global metadata persistence is added in 2.x branch. These changes will get released in 2.12 version